cpp/environment/BondOrder.cc

```diff
@@ -99,27 +89,26 @@ void BondOrder::reduceBondOrder()
                      = m_local_bin_counts.begin();
                      local_bins != m_local_bin_counts.end(); ++local_bins)
                 {
-                    m_bin_counts.get()[sa_i((int) i, (int) j)] += (*local_bins)[sa_i((int) i, (int) j)];
+                    m_bin_counts((int) i, (int) j) += (*local_bins)((int) i, (int) j);
                 }
-                m_bo_array.get()[sa_i((int) i, (int) j)]
-                    = m_bin_counts.get()[sa_i((int) i, (int) j)] / m_sa_array.get()[sa_i((int) i, (int) j)];
+                m_bo_array((int) i, (int) j)
+                    = m_bin_counts((int) i, (int) j) / m_sa_array((int) i, (int) j);
             }
         }
     });
-    Index2D sa_i = Index2D(m_n_bins_theta, m_n_bins_phi);
     for (unsigned int i = 0; i < m_n_bins_theta; i++)
     {
         for (unsigned int j = 0; j < m_n_bins_phi; j++)
         {
-            m_bin_counts.get()[sa_i((int) i, (int) j)]
-                = m_bin_counts.get()[sa_i((int) i, (int) j)] / (float) m_frame_counter;
-            m_bo_array.get()[sa_i((int) i, (int) j)]
-                = m_bo_array.get()[sa_i((int) i, (int) j)] / (float) m_frame_counter;
+            m_bin_counts((int) i, (int) j)
```

Comment:
Is it still necessary to cast to `int`?
```suggestion
            m_bin_counts(i, j)
```

Reply:
I didn't mark the rest, so please read carefully -- I didn't realize how many times the pattern was repeated.